test/FunctionalTests/Microsoft.OData.Core.Tests/Evaluation/ODataEntryMetadataContextTest.cs

```diff
@@ -209,14 +209,6 @@ public void ETagPropertiesShouldReturnEmptyForEntryMetadataContextWithoutModelAn
             this.entryMetadataContextWithoutModel.ETagProperties.Should().BeEmpty();
         }
 
-        [Fact(Skip = "we should remove this case because we need model to calc ETag")]

```

Comment:
In case as such where the test's dependency is not satisfied, should we comment out the tests (well pretty much same as using skip but w/o impacts to summary)instead of deleting it unless we are sure it is not needed anymore?